include/deal.II/distributed/grid_refinement.h

```diff
@@ -163,8 +165,9 @@ namespace parallel
       refine_and_coarsen_fixed_fraction(
         parallel::distributed::Triangulation<dim, spacedim> &tria,
         const dealii::Vector<Number> &                       criteria,
-        const double top_fraction_of_error,
-        const double bottom_fraction_of_error);
+        const double                top_fraction_of_error,
+        const double                bottom_fraction_of_error,
+        const VectorTools::NormType norm_type = VectorTools::NormType::L1_norm);
```

Comment:
Can you document the new argument?

Reply:
Documentation is missing for all arguments of this function and the corresponding `refine_and_coarsen_fixed_number` function. Their documentation refers to the non-parallel implementation in `GridRefinement`.

I will copy the `@param` documentation from the `GridRefinement` namespace for both functions.

Reply:
Thanks!